TheSyndicate/Scenes.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace TheSyndicate
+{
+    public class Scene
+    {
+        public string Id { get; set; }
```

Comment:
I think the setters should all be private, we don't want anyone to be able to modify the code: `public string Id { get; private set; }`

Reply:
Good point